packages/web-components/fast-components/src/tabs/tab/tab.styles.ts

```diff
@@ -24,25 +34,45 @@ export const TabStyles = css`
         font-weight: 400;
         line-height: 18px;
         height: calc(${heightNumber} * 1px);
-        padding: 0 calc((6 + (var(--design-unit) * 2 * var(--density))) * 1px);
-        color: var(--neutral-foreground-rest);
+        padding: calc(var(--design-unit) * 5px) calc(var(--design-unit) * 4px);
```

Comment:
Tab has this for design system support, but Tabs doesn't. We might want to keep this consistent, but I think it would be fine to not add support for calculations based on design-unit until we add density support. This will likely change a bit.

Reply:
I updated tabs to use design-unit.